.github/workflows/yehezgun.yml

```diff
@@ -0,0 +1,38 @@
+name: Template Git Actions
+
+run-name:
```

Comment:
If we don't specify a proper `run-name`, it's better to delete it.

```suggestion

```

Reply:
Deleted